app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java

```diff
@@ -394,9 +398,17 @@ public void onBindViewHolder(Presenter.ViewHolder viewHolder, Object item) {
             holder.mCardView.setPlayingIndicator(true);
         } else {
             holder.mCardView.setPlayingIndicator(false);
-            Drawable badge = rowItem.getBadgeImage();
-            if (badge != null) {
-                ((ViewHolder) viewHolder).mCardView.setBadgeImage(badge);
+
+            if (get(UserPreferences.class).get(UserPreferences.Companion.getDefaultRatingProvider()) == RatingProvider.RATING_RT) {
+                Drawable badge = rowItem.getBadgeImage();
+                ((ViewHolder) viewHolder).mCardView.setRating(null);
```

Comment:
Looks like you could use `holder` here instead of casting repeatedly 😉 .

Reply:
ah yes. it appears I started with some copy pasta and rolled with it 🤦‍♂️